recursive-list-main/src/components/breadcrumbs/breadcrumbs.component.tsx

```diff
@@ -0,0 +1,61 @@
+import { Breadcrumbs as MuiBreadcrumbs, Link } from "@material-ui/core";
+import { Link as RouterLink, useLocation } from "react-router-dom";
+import { useBreadcrumbsStyles } from "./breadcrumbs.styles";
+import { ReactComponent as SvgHome } from "../../assets/icon/home.svg";
+export interface Breadcrumb {
+  path: string;
+  name: string;
+}
+export interface GetBreadcrumbList {
+  (): Breadcrumb[];
+}
+
+const Breadcrumbs = () => {
+  const classes = useBreadcrumbsStyles();
+  const { pathname } = useLocation();
+
+  const getBreadcrumbList: GetBreadcrumbList = () => {
+    let basePath = "";
+
+    /* Returns an array of the breadcrumb list containing
+    @param name
```

Comment:
check jsDoc, it should not be written like this

Reply:
Added comment using jsDoc